pybaseball/playerid_lookup.py

```diff
@@ -60,21 +60,19 @@ def get_lookup_table(save=False):
 
 def name_similarity(last: str, first: str, player_table: pd.DataFrame) -> pd.DataFrame:
```

Comment:
what this function does has changed so the name should really change too

Reply:
I changed the name to get_closest_names_df but open to any feedback or other suggestions.